CHANGELOG.md

```diff
@@ -1,3 +1,101 @@
+## [0.3.0] - (Sep 12, 2019) ##
+
+### Added
+
+- CI build by TravisCI for ruby versions: `2.3/2.4/2.5/2.6/jruby-9.2.8.0`
+
+- fix specs with support of MacOS time formats
+
+- `Ferrum::Mouse::CLICK_WAIT` as `FERRUM_CLICK_WAIT` `ENV-var` with `0.1` as default value
+
+- `Ferrum::Browser#authorize` option `:type` with valid values `:server` (by default), `:proxy`
+
+- Logo :tada:
+
+- `Ferrum::Node#inner_text` - evaluates JS: `this.innerText` on Node instance
+
+- `Ferrum::Page::Runtime::INTERMITTENT_ATTEMPTS` as `FERRUM_INTERMITTENT_ATTEMPTS` `ENV-var` with `6` as default value
+
+- `Ferrum::Page::Runtime::INTERMITTENT_SLEEP` as `FERRUM_INTERMITTENT_SLEEP` `ENV-var` with `0.1` as default value
+
+- `Ferrum::Page#on` getting the `name` as option with `:dialog/:request_intercepted` cases & `block` as last argument
+
+- `Ferrum::Browser#on` - delegated actions to `Ferrum::Page` instance
+
+- `Ferrum::Dialog` object to handle JavaScript Dialog's
+
+    - required `page, params` as init arguments
+
+    - `#accept` fires JS: `Page.handleJavaScriptDialog` as command on page instance with options which included `accept: true`
+
+    - `#dismiss` fires JS: `Page.handleJavaScriptDialog` as command on page instance with `accept: false`
+
+    - `#match?` compare message by passed regexp
+
+    - description of `Dialog` feature in README
+
+- `Ferrum::Page::Event` extend of `Concurrent::Event` with implementation of `reset/wait` fix
+ 
+    - implement `Ferrum::Page::Event#iteration` to reuse `synchronize` block on `@iteration` value of `Concurrent::Event`
+
+    - redefinition of `Concurrent::Event#reset` - increase `@iteration` outside of `if @set` block
```

Comment:
@route this fix looks pretty good to make PR to gem `ruby-concurrency`, WDYT?

https://github.com/rubycdp/ferrum/blob/999912928ed8d105723e9edbc9c7d6a71238afb8/lib/ferrum/page.rb#L18-L30

Reply:
```
    # Reset a previously set event back to the `unset` state
    # with the synchronized increase of iteration counter
    # Has no effect on `set` if the `Event` has not yet been set.
```

Reply:
Don't really know if they need it I just wanted to see if iteration has changed but who knows maybe..